app/routers/v2.py

```diff
@@ -19,6 +19,13 @@ class Sources(str, enum.Enum):
     NYT = "nyt"
 
 
+def get_latest_data(locations):
```

Comment:
```suggestion
def _get_latest_data(locations) -> dict:
```
Can we make this private since it really shouldn't be used outside of this module?